src/pages/Home/components/ThisDay/ThisDay.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import {GlobalSvgSelector} from "../../../../assets/icons/global/GlobalSvgSelector";
+import {ThisCity, ThisDayHeader, ThisDayName, ThisTemp, ThisTime, TopBlock} from "./ThisDay_s";
+interface Props{
+}
+
+export const ThisDay =(props:Props)=>{
+  return <ThisDayHeader>
+    <TopBlock>
```

Comment:
You don't need to create every block by styled component, you can also use the css in element if  it's won't reuse and has few css
```suggestion
    <div css={{ marginBottom: '10px' }}>
```